drivers/hmis/app/models/hmis/hud/processors/service_processor.rb

```diff
@@ -43,5 +36,18 @@ def schema
 
     def information_date(_)
     end
+
+    # This post-processing step provides functionality needed for the FY2024 service form. It can be removed after FY2026 changes
+    # have been deployed.
+    #
+    # FY2024 service form collects FA Start Date for SSVF Financial Assistance, and does NOT collect Date Provided. (Needs processor to store FA Start Date in Date Provided field)
+    # FY2026 service form collects both Date Provided and FA Start Date independently for SSVF Financial Assistance
+    def post_process
+      service = @processor.send(factory_name)
+
+      # Overwrite the DateProvided field if FA Start Date was submitted and Date Provided was not
+      date_provided_submitted = @hud_values.key?('dateProvided') && @hud_values['dateProvided'] != HIDDEN_FIELD_VALUE
```

Comment:
if `dateProvided` was submitted but it's blank, should it still be treated as submitted? Maybe 
```suggestion
      date_provided_submitted = @hud_values['dateProvided'].present? && date_provided_value != HIDDEN_FIELD_VALUE
```

Reply:
Good question. I shouldn't matter in practice, because Date Provided is always required so it will be kicked back to the user if it's left blank. (I did test that case on both new and old forms). I think I'll opt to leave it as-is, since I tested it this way, and I do think we should treat it as "submitted" if its present and not hidden, regardless of the value. Again, though, it shouldn't make a difference in practice